certified-connectors/Iconfirm/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4257 @@
+          },
+          {
+            "in": "query",
+            "name": "processingDefinitionVersion",
+            "description": "The version of the processing definition used for this record.",
+            "type": "integer",
+            "format": "int32",
+            "x-ms-summary": "The version of the processing definition used for this record."
+          },
+          {
+            "in": "query",
+            "name": "nextKey",
+            "description": "A link to the next set of records.",
+            "type": "string",
+            "x-ms-summary": "A link to the next set of records."
+          },
+          {
+            "in": "query",
+            "name": "$top",
+            "description": "Only retrieve this number of records.",
+            "type": "string",
+            "x-ms-summary": "Only retrieve this number of records."
+          },
+          {
+            "in": "query",
+            "name": "$skip",
+            "description": "Skip the number of records specified here.",
+            "type": "string",
+            "x-ms-summary": "Skip the number of records specified here."
+          },
+          {
+            "in": "query",
+            "name": "$maxPageSize",
+            "description": "The number of records per page.",
+            "type": "string",
+            "x-ms-summary": "The number of records per page."
+          },
+          {
+            "in": "query",
+            "name": "$count",
+            "description": "Display result count (\"True\" or empty).",
+            "type": "string",
+            "x-ms-summary": "Display result count (\"True\" or empty)."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of records retrieved successfully",
+            "schema": {
+              "$ref": "#/definitions/PagedRecordResponse"
+            }
+          },
+          "400": {
+            "description": "Bad Request. See response body for more details"
+          },
+          "401": {
+            "description": "Invalid credentials used"
+          },
+          "501": {
+            "description": "This search functionality has not yet been implemented"
+          },
+          "503": {
+            "description": "STS Server does not respond. Unable to authenticate"
+          }
+        },
+        "operationId": "FindRecords"
+      },
+      "post": {
+        "tags": [
+          "RecordApi"
+        ],
+        "summary": "Create Record",
+        "description": "Registeres a new Record with the defined Processing Definition, ensure that all your required fields are aligned within the request with the defined processing definition.",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "description": "",
```

Comment:
@iconfirm-development Empty property values can be removed until unless you have a specific reason to keep them.

**Applies the same comment anywhere in the connector artifacts**

Reply:
Description added to the body parameter

Reply:
@iconfirm-development There are a few other empty properties exist in the swagger, Please do not resolve the comment when it is not properly resolved.